src/overlays/actors/ovl_Dm_Opstage/z_dm_opstage.h

```diff
@@ -4,23 +4,24 @@
 #include "global.h"
 
 #define DMOPSTAGE_GET_TYPE(thisx) (((thisx)->params & 0xFF))
-#define DMOPSTAGE_GET_08(thisx) (((thisx)->params >> 8) & 0xFF)
+#define DMOPSTAGE_GET_ACTORACTION(thisx) (((thisx)->params >> 8) & 0xFF)
 
 struct DmOpstage;
 
 typedef void (*DmOpstageActionFunc)(struct DmOpstage*, GlobalContext*);
+
 typedef struct DmOpstage {
     /* 0x000 */ DynaPolyActor dyna;
-    /* 0x15C */ UNK_TYPE1 pad_15C[0x44];
+    /* 0x15C */ UNK_TYPE1 pad15C[0x44];
     /* 0x1A0 */ DmOpstageActionFunc actionFunc;
     /* 0x1A4 */ Vec3f pos;
 } DmOpstage; // size = 0x1B0
 
 typedef enum {
-    /* 0 */ DM_OPSTAGE_TYPE_FLOOR,
-    /* 1 */ DM_OPSTAGE_TYPE_TREE1,
-    /* 2 */ DM_OPSTAGE_TYPE_TREE2,
-    /* 3 */ DM_OPSTAGE_TYPE_TREE3,
+    /* 0 */ DMOPSTAGE_TYPE_FLOOR,
+    /* 1 */ DMOPSTAGE_TYPE_ROOT_TREE,
+    /* 2 */ DMOPSTAGE_TYPE_CUT_TREE,
+    /* 3 */ DMOPSTAGE_TYPE_STRAIGHT_TREE,
 } DmOpStageTypes;
```

Comment:
Above the actor struct is supposed to be the standard, although I don't think we've enforced that very carefully. Also, not very keen on `FLOOR`, `GROUND` sounds better, "floor" usually means of a room in a building.

Reply:
Yeah I'm okay with that, Floor is more of a 3d space term but in this case while it is the floor its drawn as a ground surface and the trees are named on appearance in the same context.